### DISCUSSION TITLE
Next release

### DISCUSSION THREAD

**Discussion by MadsAndreasen** [asabil/vscode-meson#98] [General]
Hi, 

I was wondering if there are any thoughts on when the next release will happen?

And If there is anything I can do to help it along?

best regards,
Mads

**Comment by Ongy** [comment]
we have no process so far. It's been kind of ad hoc :)

I think when https://github.com/mesonbuild/vscode-meson/pull/94 is merged it makes sense.
Since we got the current contributions in then, and have some clear benefits over the last release.

**Reply by MadsAndreasen** [reply]
Sounds good. I would definitely benefit from getting 94 in 👌

**Reply by dcbaker** [reply]
I’ll plan to make the release today then

**Comment by dcbaker** [comment]
And Done

**Reply by MadsAndreasen** [reply]
Great. 🤗
Thanks
